### ISSUE TITLE
When assert the same field more than once, only one assert is executed.

### ISSUE THREAD

**Issue by chonla** [chonla/cotton#12]
Refer to: https://github.com/chonla/cotton/issues/10#issuecomment-414303315

When assert the same field more than once, only one assert is executed.

For example:

```
| Data.d.results[0].Region | *Should be null* |
| Data.d.results[0].Region | *Should exist* |
```

Then only either of them is asserted but not all.

**Comment by chonla** [comment]
Fixed in af4850ddd55bb610b8569e93bed1bf5d0fe9fee4.

**Comment by pacroy** [comment]
Looks great. Thank you!
